radixdlt/src/main/java/org/radix/Radix.java

```diff
@@ -32,6 +33,7 @@
 import com.radixdlt.universe.Universe;
 import com.radixdlt.utils.Bytes;
 
+import io.reactivex.rxjava3.observables.ConnectableObservable;
```

Comment:
I don't think this import is used, but checkstyle didn't seem to complain about it?

Reply:
yeah it doesn't seem like Radix.java is being checked